.internal-ci/util/generate_dev_values-v5.1.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Add a generate_dev_values for new charts and start pulling values out to make actions easier to follow?